src/handlers/handleFinish.ts

```diff
@@ -34,21 +33,31 @@ export default function handleFinish(
   }
 
   // Print the security report
-  if (report.length) {
-    printSecurityReport(report, columnsToInclude);
-  }
+  if (outputJson) {
+    printJsonOutput(result, exceptionsReport);
```

Comment:
with json output we only print the json, any other logs should be suppressed so the output can be parsed as json